packages/react/src/PricingOptions/PricingOptions.module.css

```diff
@@ -0,0 +1,339 @@
+/* ---------------------------------------------------------- */
+
+.PricingOptions--presentation-cards .PricingOptions__item {
+  padding-inline: var(--base-size-32);
+  border-radius: var(--brand-borderRadius-xlarge);
+  border: solid 1px var(--brand-color-border-default);
+  background-color: var(--brand-color-canvas-default);
+  padding-block: var(--base-size-32);
+}
+
+/* ---------------------------------------------------------- */
+/* 4. Single item                                             */
+/* ---------------------------------------------------------- */
+
+@media (min-width: 63.25rem) {
+  .PricingOptions--items-1 {
+    --pricing-cards-container-padding-inline: var(--base-size-48);
+
+    display: block;
+  }
+
+  .PricingOptions--items-1 .PricingOptions__item {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: calc(var(--base-size-96) * 2);
+    grid-template-rows: repeat(auto-fit, minmax(0, 1fr));
+    grid-template-areas:
+      'label actions'
+      'heading actions'
+      'description actions'
+      'price actions'
+      'feature-list actions';
+  }
+
+  .PricingOptions--items-1 .PricingOptions__actions {
+    align-self: center;
+  }
+}
+
+/* ---------------------------------------------------------- */
+/* 5. Multiple items                                          */
+/* ---------------------------------------------------------- */
+
+@media (min-width: 63.25rem) {
+  .PricingOptions--items-2,
+  .PricingOptions--items-3 {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
+    column-gap: var(--pricing-cards-item-gap);
+    row-gap: 0;
+
+    grid-template-areas:
+      'label'
+      'heading'
+      'description'
+      'price'
+      'actions'
+      'feature-list'
+      'footnote';
+  }
+
+  .PricingOptions--presentation-default.PricingOptions--items-2 {
+    --pricing-cards-item-gap: var(--brand-PricingOptions-items-2-gap);
+    --pricing-cards-container-padding-inline: var(--brand-PricingOptions-items-2-container-padding-inline);
+  }
+
+  .PricingOptions--presentation-default.PricingOptions--items-3 {
+    --pricing-cards-item-gap: var(--brand-PricingOptions-items-3-gap);
+    --pricing-cards-container-padding-inline: var(--brand-PricingOptions-items-3-container-padding-inline);
+  }
+}
+
+/* ---------------------------------------------------------- */
+/* 6. Item inner layout                                       */
+/* ---------------------------------------------------------- */
+
+@media (min-width: 63.28rem) {
+  .PricingOptions__item {
+    display: grid;
+    grid-row: label / footnote;
+    grid-template-rows: subgrid;
+  }
+}
+
+/*
+ * Backwards compatibility for browsers that don't support subgrid.
+ * Once subgrid is supported in the last 3 major versions of Safari (i.e., when Safari 18 is released), it can be fully removed.
+ * https://caniuse.com/css-subgrid
+ */
+@supports not (grid-template-rows: subgrid) {
```

Comment:
Can this be done as a progressive enhancement, where using subgrid is optional and opt-in? Looks like it's the other way around right now, which might not fly with house rules. That said, this is much nicer for balancing our tech debt.

Reply:
I implemented the progressive enhancement approach initially but chose this route because: 
1. It's as safe as the progressive enhancement: the `@supports` at-rule is supported on all our supported browsers 🤪, which ensures everyone is getting the "baseline" behavior
2. It is much simpler and easier to read than the progressive enhancement implementation. And we can delete the whole statement when `subgrid` is fully supported.